### PR TITLE
Updated GitIgnore.

### PR DESCRIPTION
**Pull request by ruehyder**
Creating a Pull Request

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,195 @@
+## Ignore Visual Studio temporary files, build results, and
+## files generated by popular Visual Studio add-ons.
+
+# User-specific files
 *.suo
-*.lock
```
Comment (Bhusting) [inline .gitignore:2]: Test

File: .gitignore [hunk 2]
```diff
@@ -1,5 +1,195 @@
+## Ignore Visual Studio temporary files, build results, and
```
Comment (ruehyder) [inline .gitignore:1]: Commenting works

File: .gitignore [hunk 3]
```diff
@@ -1,5 +1,195 @@
...
+*.vssscc
+.builds
+*.pidb
+*.svclog
+*.scc
+
+# Chutzpah Test files
+_Chutzpah*
+
+# Visual C++ cache files
```
Comment (swynnr) [inline .gitignore:64]: Test

File: .gitignore [hunk 4]
```diff
@@ -1,5 +1,195 @@
...
+
+# Guidance Automation Toolkit
+*.gpState
+
+# ReSharper is a .NET coding add-in
+_ReSharper*/
+*.[Rr]e[Ss]harper
+*.DotSettings.user
+
+# JustCode is a .NET coding addin-in
```
Comment (swynnr) [inline .gitignore:88]: Test 2

File: .gitignore [hunk 5]
```diff
@@ -1,5 +1,195 @@
...
-*.ide-wal
\ No newline at end of file
+*.user
+*.sln.docstates
+*.Rli.Runtime.Api.DESKTOP-4EPNO80-configuration.txt
+
+# Build results
+[Dd]ebug/
+[Dd]ebugPublic/
+[Rr]elease/
```
Comment (ruehyder) [inline .gitignore:13]: Comment testing

File: .gitignore (hunk 1 above)
Comment (nollanr1) [reply .gitignore:2]: This is a comment

**Review by ruehyder — COMMENTED** [review]
Comment testing

File: .gitignore (hunk 1 above)
Comment (swynnr) [reply .gitignore:2]: Yeet

File: .gitignore [hunk 6]
```diff
@@ -1,5 +1,195 @@
+## Ignore Visual Studio temporary files, build results, and
+## files generated by popular Visual Studio add-ons.
```
Comment (swynnr) [inline .gitignore:2]: Test

**Pull request merged**